index.d.ts

```diff
@@ -77,6 +86,25 @@ export const init: (config: InitConfig) => void
 
 export const getFirebaseAdmin: () => app.App
 
+/**
+ * Get the Firebase Client API. Use this when developing on the Client (Browser).
+ * Before usage ensure that each of the Firebase Modules required are imported into
+ * the project (See https://firebase.google.com/docs/web/setup#available-libraries); for example add:
+ *
+ * @example
+ * ```
+ * // Add the Firebase products that you want to use (note that this module imports `firebase/auth` already.
+ * import "firebase/firestore";
+ * import "firebase/functions";
+ * import "firebase/messaging";
+ * import "firebase/analytics";
+ * import "firebase/storage";
+ * import "firebase/database";
+ * ```
+ *
+ */
+export const getFirebaseClient: () => app.App

```

Comment:
I would expect this type to be different from `getFirebaseAdmin`'s type. Should `getFirebaseAdmin` use types from the `firebase-admin` app instead?

Reply:
Fixed in commit; thank you for spotting that!!